ledger-service/http-json/src/main/scala/com/digitalasset/http/HttpService.scala

```diff
@@ -100,7 +101,7 @@ object HttpService extends StrictLogging {
     val tokenHolder = accessTokenFile.map(new TokenHolder(_))
 
     val clientConfig = LedgerClientConfiguration(
-      applicationId = ApplicationId.unwrap(applicationId),
+      applicationId = ApplicationId.unwrap(DummyApplicationId),
```

Comment:
Maybe worth adding a comment here that this is only used by the asynchronous command service?

Reply:
I don't like comments like this for one reason... there is no way to enforce the correctness of such comments, they tend to become wrong over time :) the name with the dummy prefix assumes that it is Noop, I can rename it to NoopApplicationId. Unless you insist on adding the comment.

Reply:
your choice, personally I’ve gained far more from having useful comments than I’ve wasted due to out of date comments so I tend to prefer adding a comment (useful comments of course, I’m not a fan of the “document every parameter and variable everywhere” approach) but I don’t feel strongly in this particular case.